framework/include/actions/AddChainControlAction.h

```diff
@@ -0,0 +1,23 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "AddControlAction.h"
+
+/**
+ * Adds a ChainControl.
```

Comment:
I'll do "to the control warehouse". The ChainControlSystem doesn't come into play until those ChainControls start adding their data. It might be better named "ChainControlDataSystem" if people think that's appropriate.

Reply:
Makes sense